### PR TITLE
framework updated to dotnet6

### PR DESCRIPTION
**Pull request by mheidari988**

### CODE REVIEW

File: Pages/Index.cshtml [hunk 1]
```diff
@@ -6,5 +6,5 @@
 
 <div class="text-center">
     <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
+    <p>Learn about <a href="#">building Web apps with ASP.NET Core Blazor</a>.</p>
```
Comment (fxtr0) [inline Pages/Index.cshtml]: Link is broken here. Did you checked out?

**Comment by fxtr0** [comment]
I see there is some changes beside framework change. Do we need more changes ?

**Comment by mheidari988** [comment]
checking

**Review by fxtr0 — CHANGES_REQUESTED** [review]
Check the broken link

**Review by fxtr0 — APPROVED** [review]
done

**Pull request merged**